generate.sh

```diff
@@ -11,12 +11,12 @@ if [[ -z "${TT_REPO}" ]]; then
 	TT_REPO="https://github.com/tarantool/tarantool.git"
 fi
 
-TT_DIR=tarantool
+TT_DIR=tarantool-$TT_TAG
```

Comment:
Just to be with the same style.
```suggestion
TT_DIR=tarantool-${TT_TAG}
```